web3/eth_api_types.nim

```diff
@@ -139,21 +164,29 @@ type
     error*: Opt[string]
     gasUsed*: Quantity
 
-  TransactionObject* = ref object                    # A transaction object, or null when no transaction was found:
-    hash*: TxHash                                    # hash of the transaction.
-    nonce*: Quantity                                 # TODO: Is int? the number of transactions made by the sender prior to this one.
+  AuthorizationObject* = object
+    chainId*: Quantity
+    address*: Address
+    nonce*: Quantity
+    yParity*: Quantity
+    R*: UInt256
+    S*: UInt256
```

Comment:
Are you sure that these are the names used on JSON-RPC? in the transaction it's lowercase `r`/`s`, and `yParity` is `v`.

Reply:
right, they should be `v', 'r', 's', all in lowercase.